force-app/main/default/objects/TemplateTask__c/fields/Task__c.field-meta.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>Task__c</fullName>
```

Comment:
"Be careful when changing the label as it may affect existing merge templates." this is a popup message when I tried to change it, do I still go ahead or will this be an issue in the future?

Reply:
Yes. Merge templates relates to email templates, it won't cause any issues in the future because it only applies to the existing ones.

Reply:
Got it thanks

Reply:
Complete